error.log

```diff
@@ -0,0 +1 @@
+{"level":"error","message":"Invalid authorization token","service":"user-service","timestamp":"2020-09-05T17:00:01.596Z"}
```

Comment:
Please, add this file to the `.gitignore`, logs should not be pushed to the backend